<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestList</class>
 <widget class="QDialog" name="TestList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
font: 18pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Created tests</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="subjectLabel">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Subject:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="subjectComboBox">
           <item>
            <property name="text">
             <string>Fizica</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Matematica</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Filosofie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Geografie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Romana</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setSubjectButton">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Assign test to:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="assignGroupsList">
           <item>
            <property name="text">
             <string>C112A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C112D</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C112E</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A112B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C112JMK</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="assignButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Assign</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="testsTableList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createTestButton">
       <property name="text">
        <string>Create new test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>TestList</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
